JenkinsFile

```diff
@@ -0,0 +1,25 @@
+pipeline {
```

Comment:
_This review comment includes notes on the way you wrote up you commit messages_

- have you checked how commit messages are really written, have you read git commit message semantics?
- do we really write commit messages as past sentences?
- how your first commit on this PR makes sense?

Reply:
readed git commit message semantics, will work according to that from next time